### PR TITLE
Add manually added/substracted goal stats in frontend

### PR DESCRIPTION
**Pull request by KacperMalachowski**
## Summary of changes

1. Added rows to the game statistics table for displaying added or subtracted goals per each team.
2. Added three functions (we should consider extracting them from the GameStatistics component later) that encapsulate logic for formatting data from the component state.

### CODE REVIEW

**Review by MichalKalke — APPROVED** [review]
LGTM

File: frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js [hunk 1]
```diff
@@ -46,6 +59,12 @@ function GameStatistics({ finalScores, onNewGameRequested }) {
         <div className="table-item">{returnFastestShot(TeamID.Team_blue)}</div>
         <div className="table-item">fastest shot of the game</div>
         <div className="table-item">{returnFastestShot(TeamID.Team_white)}</div>
+        <div className="table-item">{getManualAddedGoals(TeamID.Team_blue)}</div>
+        <div className="table-item">Manually added goals</div>
+        <div className="table-item">{getManualAddedGoals(TeamID.Team_white)}</div>
+        <div className="table-item">{getManualSubstractedGoals(TeamID.Team_blue)}</div>
+        <div className="table-item">Manually substracted goals</div>
+        <div className="table-item">{getManualSubstractedGoals(TeamID.Team_white)}</div>
```
Comment (Lyczeq) [inline frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js:67]: Consider (now or later) to create some abstraction for these divs. You will have more and more statistics, so the list of divs will grow as well. That won't look good.

**Review by Lyczeq — APPROVED** [review]

File: frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js [hunk 2]
```diff
@@ -21,6 +21,19 @@ function GameStatistics({ finalScores, onNewGameRequested }) {
     return Team === teamID ? Speed.toFixed(2) + ' km/h' : '😵';
   }
 
+  function getManualChangedGoals(teamID) {
+    if (!statistics?.ManualGoals) return;
+    return statistics.ManualGoals[teamID];
```
Comment (marekkawalski) [inline frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js:26]: Endpoint has changed.
```suggestion
    if (!statistics?.manualGoals) return;
    return statistics.manualGoals[teamID];
```

**Review by marekkawalski — APPROVED** [review]

File: frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js (hunk 1 above)
Comment (KacperMalachowski) [reply frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js:67]: For now, I leave it as is. It will be fixed in separate PR

**Pull request merged**